.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.12
+      - image: circleci/node:11
```

Comment:
@tmeasday I was only able to get this working w/ Node 11. Some sort of nested dependency issue with Node 12 was sending me into a rabbit hole.